docs/source/usage_guides/model_adaptation.mdx

```diff
@@ -0,0 +1,83 @@
+<!---
+Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# Model Adaptation
```

Comment:
Still not convinced by this naming. "GPU migration" sounds good to me as used in the Gaudi doc here: https://docs.habana.ai/en/latest/PyTorch/PyTorch_Model_Porting/GPU_Migration_Toolkit/GPU_Migration_Toolkit.html
WDYT?

Reply:
> Still not convinced by this naming. "GPU migration" sounds good to me as used in the Gaudi doc here: https://docs.habana.ai/en/latest/PyTorch/PyTorch_Model_Porting/GPU_Migration_Toolkit/GPU_Migration_Toolkit.html WDYT?

This one is a but sticky.   I think that Model Migration refers to process of performing Migration from general purpose GPU to Gaudi HPU.   However, in OH, we kind of migrated this by providing Gaudi specific pipeline.  So the user kind of needs to just "adapt" a code from HF libs (diffusers, transformers, ..) to OH in that sense.  Like instead of StableDiffuseion pipeline adapt and use GaudiStableDiffusion (didn't actually perform model migration).   It is my understanding but maybe I am totally off here :)   Your thoughts?

Reply:
The way I see it, this page shows how to port a Transformers/Diffusers script from GPU to HPU.
IMO "model adaptation" could mean many different things and users don't really have to know how different models are to use them on Gaudi. That's why any title with "model" sounds a bit odd to me.
Maybe this could be called "Adapt a Transformers/Diffusers script to Intel Gaudi". This is a bit verbose but at least much clearer regarding the content of this page.
But anyway let's merge this PR today and we can modify this later if we don't find something good here.

Reply:
Ok I think you nailed the issue.  Title does come across as confusing and could be misleading as different people may understand it differently.  I will change it.

The actual Model Migration page would be actually describing steps for model migration, like taking existing say transformers or diffusers pipeline and migrating to Gaudi.  Maybe something along these lines can be added in future (for more advanced, daredevil developers :)

Reply:
Fixed the title just now.  Also fixed the overview to better convey the pages organization in `usage_guides` section